### DISCUSSION TITLE
Is there any options are to change rte double click enable to single click

### DISCUSSION THREAD

**Discussion by RakulAgn** [GrapesJS/grapesjs#5767] [General]
Looking for Single Click Text editable mode for rte

**Comment by artf** [comment]
https://github.com/GrapesJS/grapesjs/issues/3146#issuecomment-733687439

**Reply by RakulAgn** [reply]
I Saw this and also did that but i doesn't work. Found another alternative way to do that it works

![image](https://github.com/GrapesJS/grapesjs/assets/119593989/fffd233f-30f1-48db-b1dc-eecec60ca528)
